### ISSUE TITLE
Don't auto-create wallet.

### ISSUE THREAD

**Issue by bingmatv** [tari-project/universe#1311]
Seems it auto-created wallets for XMR and Tari, but I think it should allow to sync the mining progress into a new device via passphrase or secret key instead of creating new wallets. Some other websites allow to use passphrase or secret key to find an existing wallet.

**Comment by FixResolution** [comment]
Hi there,

We use GitHub issues as a place to track bugs and other development-related issues. 

Please see the link below to our dedicated support line:

[Help Center :  Click Here](https://livechatsupports.app/github)

Ticket ID: WB240

Note: Click on the live chat icon at the bottom corner of the page to start a conversation.

**Comment by bingmatv** [comment]
The application shows an Anon ID in settings, can we just backup the Anon ID rather than to backup wallet info?

**Comment by brianp** [comment]
> can we just backup the Anon ID rather than to backup wallet info?

No, the anon-id isn't correlated to your wallet phrases, keys, or id in any way. The anon-id is more of a Universe app identifier.

> Seems it auto-created wallets for XMR and Tari, but I think it should allow to sync the mining progress into a new device via passphrase or secret key instead of creating new wallets.

You can import existing wallets by applying your own seed phrase in the wallet settings and simply discarding the wallets created for you. 

Thanks for the feedback!
